src/utilities.jl

```diff
@@ -1,7 +1,7 @@
 macro inboundsret(ex)
     return quote
         @inbounds v = $(esc(ex))
-        v
+        return v
```

Comment:
Thanks for this ;) This only worked if `@inboundsret` was the last expression in the function...